docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+services:
+  metabase:
+    image: metabase/metabase-enterprise:v1.52.x
+    environment:
+      MB_CONFIG_FILE_PATH: "./app/metabase/init-config.yml"
+      MB_JETTY_PORT: "${MB_PORT}"
+      MB_EDITION: "ee"
+      MB_EMBEDDING_APP_ORIGIN: "*"
+      MB_ENABLE_EMBEDDING: true
+      MB_SITE_URL: "http://localhost:${MB_PORT}/"
+      MB_JWT_ENABLED: true
```

Comment:
(FYI, I think these probably can be specified via `init-config.yml` as well, but this is also totally ok.)

Reply:
Moved this and MB_ENABLE_EMBEDDING